Services/Excel/classes/class.ilExcel.php

```diff
@@ -351,7 +351,8 @@ public function getColumnCount() {
 	 */
 	public function getColumnCoord($a_col)
 	{
-		return Coordinate::stringFromColumnIndex($a_col);
+		$col = ++$a_col;
+		return Coordinate::stringFromColumnIndex($col);
```

Comment:
@bheyser / @xus : `return Coordinate::stringFromColumnIndex($col + 1);` would be much easier to understand :-)

Reply:
`return Coordinate::stringFromColumnIndex($a_col + 1);` :-)

Reply:
In the first place i did think that xus is the maintainer and i followed his unique style of writing the things after studying his commit to fix issues ;-) in general I agree ;-)